common.go

```diff
@@ -6,10 +6,11 @@ package saltpack
 import (
 	"bytes"
 	"crypto/hmac"
-	"crypto/rand"
+	cryptorand "crypto/rand"
```

Comment:
I did it here out of necessity (since I use both) but yeah maybe it's a good idea in general.